### ISSUE TITLE
Hello Amazing people! FragPipe 19.1 gives me a lot of weird words about Java stuff when I hit run

### ISSUE THREAD

**Issue by orsburn** [Nesvilab/FragPipe#1004]
- Describe the issue or question:

Hi Fengchao, Dr. Nevzvizhviskikzi et al., I just did an upgrade to FragPipe 19.1 and did MSFragger 3.7, the newest IonQuant and it looks happy until I hit the run button then it generates a big window complaining about Java things. 
I assumed that writing you here would be faster than reading. 
Thanks!! 

- Upload your log file
(If a log file hasn't been generated, go to the 'Run' tab in FragPipe, click 'Export Log', zip the resulting "log_[date_time].txt" file to avoid truncation, then attach the zipped file by drag & drop here.)
![image](https://user-images.githubusercontent.com/39571544/218499296-092223a9-5fbd-4430-9caa-dd1be3d40740.png)

**Comment by fcyu** [comment]
I think it is something related to the Philosopher binary path. Could you click "clear cache and close" button and try again?
![image](https://user-images.githubusercontent.com/6926299/218500500-335ec496-a808-49d9-b09f-508538d6623b.png)

Thanks,

Fengchao

**Comment by orsburn** [comment]
Wooohooo! That appeared to do it, I did have to directly download the newest Philosopher. I did the upgrade through FragPipe, and it said it was okay but I think my security stuff blocked the executable download and installation.
Strangely, this appears to be the same issue on every PC we've attempted the upgrade on, which might just be our IT security protocols.
THANK YOU! 
![thank-you-thank-u](https://user-images.githubusercontent.com/39571544/218504391-237a68cd-789f-4088-83aa-88fdbb1b60b2.gif)
